.github/workflows/manubot.yaml

```diff
@@ -0,0 +1,40 @@
+name: Manubot
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    name: Build Manuscript
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Install Environment
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: manubot
+          environment-file: build/environment.yml
+          auto-activate-base: false
+          miniconda-version: 'latest'
+      - name: Build Manuscript
+        env:
+          GITHUB_PULL_REQUEST_SHA: ${{ github.event.pull_request.head.sha }}
+        shell: bash --login {0}
+        run: bash build/build.sh
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: output
+          path: output
+      - name: Deploy
```

Comment:
Maybe `Deploy Manuscript` to keep in line with the `[verb] [noun]` pattern of the other step names above.

Reply:
Yes, I like this suggestion too